### PR TITLE
DOCS-10337 cookies (Apr 15)

### PR DESCRIPTION
**Pull request by wwarner-salesforce**
DOCS-10337 cookies (Apr 15)

Reverts mulesoft/docs-exchange#131

### CODE REVIEW

File: modules/ROOT/pages/about-administration-tasks.adoc [hunk 1]
```diff
@@ -3,23 +3,26 @@ ifndef::env-site,env-github[]
...
 endif::[]
 
-Exchange administrators can perform the following tasks:
+Exchange administrators have permissions to perform these tasks:
 
-* xref:to-set-permissions.adoc[Set Exchange role permissions]
+* xref:to-share-an-asset.adoc[Share an asset]
 +
-Determine who in your organization should be an Exchange administrator, contributor, or viewer. You can also provide role access for individual assets.
+Grant access to an asset to a user in your organization, to everyone in your organization, or to everyone in a chosen external organization, or publish an API asset to a public portal.
```
Comment (priyaayyar) [inline modules/ROOT/pages/about-administration-tasks.adoc]:
```suggestion
You can grant specific users access to an asset in your organization, to everyone in your organization, to everyone in a chosen external organization, or publish an API asset to a public portal.
```

File: modules/ROOT/pages/about-administration-tasks.adoc [hunk 2]
```diff
@@ -3,23 +3,26 @@ ifndef::env-site,env-github[]
...
 +
-This enables you to specify which API assets can be shared with users.
+Specify which API assets can be shared with users.
 * xref:to-deprecate-asset.adoc[Deprecate an asset]
 +
-Remove access to a version within an asset but keep the asset available. Users can access other versions within the asset.
+Remove access to a version of an asset but keep other versions of the asset available to users.
+* xref:to-set-permissions.adoc[Set Exchange role permissions]
++
+Grant Exchange administrator, contributor, or viewer permissions to members of your organization, and optionally provide role access for individual assets.
```
Comment (priyaayyar) [inline modules/ROOT/pages/about-administration-tasks.adoc]:
```suggestion
You can grant?? Exchange Administrator, Exchange Contributor??, or Exchange Viewer?? permissions to members of your organization, and optionally provide role access for individual assets.
```
Provide role access--is this correct?

File: modules/ROOT/pages/cookies.adoc [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+Anypoint Exchange uses cookies to save user settings, enhance the user experience, and improve performance.
+
+In general, websites use multiple types of cookie:
+
+* Required cookies necessary for the site to work correctly
+* Functional cookies necessary for certain site functionality
+* Functional cookies that are not necessary for any site functionality
+* Advertising cookies
+
+Exchange requires some cookies to work correctly, including cookies for session details and authentication information.
```
Comment (priyaayyar) [inline modules/ROOT/pages/cookies.adoc:16]:
```suggestion
Exchange requires cookies to work correctly, including cookies for session details and authentication information.
```
Does Exchange require only some cookies to work correctly? The others can work incorrectly and it'll still be OK?

File: modules/ROOT/pages/cookies.adoc [hunk 4]
```diff
@@ -0,0 +1,69 @@
...
+
+Exchange normally uses some functional cookies, including cookies for site analytics. Exchange does not require any functional cookies for site functionality.
```
Comment (priyaayyar) [inline modules/ROOT/pages/cookies.adoc]:
```suggestion
Exchange normally uses functional cookies, including cookies for site analytics. Conversely, Exchange does not require any functional cookies for site functionality.
```

File: modules/ROOT/pages/cookies.adoc [hunk 5]
```diff
@@ -0,0 +1,69 @@
...
+
+Content in Exchange documentation often uses advertising cookies. If a video embedded in an Exchange portal is hosted on a third-party site such as YouTube that uses advertising cookies, then the video is not visible to a website visitor who disables advertising cookies, and the visitor sees a message explaining this.
+
+== Cookie Consent Management
+
+Exchange provides public portal website visitors with cookie consent management and full documentation of all cookies, in compliance with marketing and privacy regulations.
+
+Some countries require that websites use non-essential cookies only after the website visitor has granted explicit consent.
+
+When a user in a country with this requirement visits an Exchange public portal for the first time, Exchange notifies the user about the cookies stored in the browser, and the user accepts the default settings or opens the cookie consent manager to configure their cookie settings. Exchange finds the user's country by IP address.
```
Comment (priyaayyar) [inline modules/ROOT/pages/cookies.adoc:28]: Cookie Consent Manager??

File: modules/ROOT/pages/portal-vanity-domain.adoc [hunk 6]
```diff
@@ -119,6 +119,30 @@ NOTE: Your server must run using `https`.
 
 If set up correctly, you can access and navigate to `+https://developers.robocop.com/path/of/justice/+` exactly as if you were at `+https://anypoint.mulesoft.com/exchange/portals/robo-cop/+`.
 
+== Configure Cookie Consent Management for Your Domain
+
+Exchange public portals that do not use vanity domains can use the xref:cookies.adoc[cookie consent manager] provided.
```
Comment (priyaayyar) [inline modules/ROOT/pages/portal-vanity-domain.adoc]:
```suggestion
Exchange public portals that do not use vanity domains can use the xref:cookies.adoc[cookie consent manager] that is provided.
```
OR 
```suggestion
Exchange public portals that do not use vanity domains can use the provided xref:cookies.adoc[cookie consent manager].
```

I'm not sure what is provided? The consent manager? Or cookies?

File: modules/ROOT/pages/portal-vanity-domain.adoc [hunk 7]
```diff
@@ -119,6 +119,30 @@ NOTE: Your server must run using `https`.
...
+
+After your vanity domain is configured and your site is working, activate cookie consent management by contacting customer support with this information:
+
+* Your organization domain (Example: `robo-cop`)
+* Your registered domain (Example: `developers.robocop.com`)
+
+Customer support provides you a `COOKIE_CONSENT_ID` for the recently registered vanity domain. This `COOKIE_CONSENT_ID` cannot be transferred to other vanity domains.
```
Comment (priyaayyar) [inline modules/ROOT/pages/portal-vanity-domain.adoc]: Passive voice..
```suggestion
Customer support provides you a `COOKIE_CONSENT_ID` for the recently registered vanity domain. You cannot transfer this `COOKIE_CONSENT_ID` to other vanity domains.
```

File: modules/ROOT/pages/portal-vanity-domain.adoc [hunk 8]
```diff
@@ -119,6 +119,30 @@ NOTE: Your server must run using `https`.
...
+
+=== Register Your Cookie Consent ID
+
+You must register your `COOKIE_CONSENT_ID`. Use the same token you obtained when you registered your domain.
```
Comment (priyaayyar) [inline modules/ROOT/pages/portal-vanity-domain.adoc]:
```suggestion
You must register your `COOKIE_CONSENT_ID` by using the same token you obtained when you registered your domain.
```

**Review by priyaayyar — CHANGES_REQUESTED** [review]

File: modules/ROOT/pages/about-administration-tasks.adoc (hunk 1 above)
Comment (wwarner-salesforce) [reply modules/ROOT/pages/about-administration-tasks.adoc]: Kim's style guide at https://wiki.corp.mulesoft.com/display/MTDT/MuleSoft+Writing+Style+Reference says not to use modal verbs, especially "can," "may," and "might," so I reworded this to improve the sentence without modal verbs.

File: modules/ROOT/pages/about-administration-tasks.adoc (hunk 2 above)
Comment (wwarner-salesforce) [reply modules/ROOT/pages/about-administration-tasks.adoc]: Yes, this provides role access. Good catch on the capitalization! Fixed.

File: modules/ROOT/pages/cookies.adoc (hunk 3 above)
Comment (wwarner-salesforce) [reply modules/ROOT/pages/cookies.adoc:16]: This is correct: Users can use the cookie consent manager to block some cookies, and Exchange will still work correctly.

File: modules/ROOT/pages/cookies.adoc (hunk 4 above)
Comment (wwarner-salesforce) [reply modules/ROOT/pages/cookies.adoc]: Fixed.

File: modules/ROOT/pages/cookies.adoc (hunk 5 above)
Comment (wwarner-salesforce) [reply modules/ROOT/pages/cookies.adoc:28]: The OneTrust site uses "OneTrust cookie consent," so we're following that here and using "cookie consent manager" as a lowercased generic term describing the functionality, and not as a capitalized product name.

File: modules/ROOT/pages/portal-vanity-domain.adoc (hunk 6 above)
Comment (wwarner-salesforce) [reply modules/ROOT/pages/portal-vanity-domain.adoc]: The consent manager is provided. Reworded to clarify.

File: modules/ROOT/pages/portal-vanity-domain.adoc (hunk 7 above)
Comment (wwarner-salesforce) [reply modules/ROOT/pages/portal-vanity-domain.adoc]: Fixed.

File: modules/ROOT/pages/portal-vanity-domain.adoc (hunk 8 above)
Comment (wwarner-salesforce) [reply modules/ROOT/pages/portal-vanity-domain.adoc]: Fixed.

**Review by priyaayyar — APPROVED** [review]

**Pull request merged**